x-pack/test/functional/apps/security/rbac_phase1.js

```diff
@@ -0,0 +1,135 @@
+/*
+* Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+* or more contributor license agreements. Licensed under the Elastic License;
+* you may not use this file except in compliance with the Elastic License.
+*/
+
+import expect from 'expect.js';
+import { indexBy } from 'lodash';
+export default function ({ getService, getPageObjects }) {
+
+  const PageObjects = getPageObjects(['security', 'settings', 'monitoring', 'discover', 'common', 'reporting', 'visualize', 'header']);
```

Comment:
nit: the `monitoring`, `discover` and `reporting` page objects aren't used anymore and can be removed.

Reply:
++ oversight. Removed the page objects which weren't necessary.